tests/components/trafikverket_weatherstation/__init__.py

```diff
@@ -1 +1,9 @@
 """Tests for the Trafikverket weatherstation integration."""
+
+from homeassistant.components.trafikverket_weatherstation.const import CONF_STATION
+from homeassistant.const import CONF_API_KEY
+
+ENTRY_CONFIG = {
+    CONF_API_KEY: "1234567890",
+    CONF_STATION: "Arlanda",
+}
```

Comment:
Isn't this something for const.py

Reply:
Still think it's quite unnecessary to define a `const.py` file in tests.